test/dotnet.Tests/project.json

```diff
@@ -43,6 +43,7 @@
     ]
   },
   "buildOptions": {
+    "keyFile": "../../tools/Key.snk",
```

Comment:
Seems to be ad hoc.

https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.ProjectModel.Tests/project.json#L4